recipes/libdivide/all/conanfile.py

```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, tools
+import os
+
+
+class LibdivideConan(ConanFile):
+    name = "libdivide"
+    description = "Header-only C/C++ library for optimizing integer division."
+    topics = ("conan", "libdivide", "division", "integer")
+    license = ["Zlib", "BSL-1.0"]
+    homepage = "http://libdivide.com/"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "compiler"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+    def package_id(self):
+        self.info.header_only()
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename(self.name + "-" + self.version, self._source_subfolder)
+
+    def package(self):
+        self.copy("LICENSE.txt", dst="licenses", src=self._source_subfolder)
+        self.copy("libdivide.h", dst="include", src=self._source_subfolder)
```

Comment:
```suggestion
        self.copy("libdivide.h", dst="include", src=self._source_subfolder)

    def package_info(self):
        if self.options.libdivide_sse2:
            self.cpp_info.defines.append("LIBDIVIDE_SSE2")
        if self.options.libdivide_avx2:
            self.cpp_info.defines.append("LIBDIVIDE_AVX2")
        if self.options.libdivide_avx512:
            self.cpp_info.defines.append("LIBDIVIDE_AVX512")

```

Reply:
What @uilianries  said + indent extra with:
```python
if self.settings.arch in ("x86", "x86_64"):
    xxx
```

Reply:
addressed in https://github.com/conan-io/conan-center-index/pull/3315/commits/b42f1247d8264b444094cdd93f0ac1fac6df3a3f